<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtlMovieAudioTestDialog</class>
 <widget class="QtlDialog" name="QtlMovieAudioTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtlMovie Audio Tracks Test</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtlMovie.qrc">
    <normaloff>:/images/qtlmovie-64.png</normaloff>:/images/qtlmovie-64.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSlider" name="volumeSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSlider" name="playSlider">
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="audioStreamsScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QtlButtonGrid" name="audioStreamsBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>156</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="volumeLabel">
     <property name="text">
      <string>Volume:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="playStopButton">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtlDialog</class>
   <extends>QDialog</extends>
   <header>QtlDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtlButtonGrid</class>
   <extends>QWidget</extends>
   <header>QtlButtonGrid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtlMovie.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieAudioTestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playStopButton</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieAudioTestDialog</receiver>
   <slot>startStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtlMovieAudioTestDialog</receiver>
   <slot>updateVolume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStop()</slot>
  <slot>updateVolume(int)</slot>
 </slots>
</ui>
